ontology/termit-glosář.ttl

```diff
@@ -385,9 +385,9 @@ termit-pojem:omezený-uživatel-termitu
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 termit:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
-                "Restricted TermIt user"@en , "Omezený uživatel TermItu"@cs ;
+                "Reader"@en , "Čtenář"@cs ;
         <http://www.w3.org/2004/02/skos/core#scopeNote>
-                "User with restricted access to TermIt. Can view existing vocabularies and terms and comment on terms."@en , "Uživatel s omezeným přístupem k systému TermIt. Může prohlížet slovníky a pojmy a komentovat pojmy."@cs .
+                "She can list and view vocabularies and terms. She can also comment on terms."@en , "Může prohlížet slovníky a pojmy. Pojmy může rovněž komentovat."@cs .
```

Comment:
Also why only She? Lets use He or She

Reply:
@MichalMed and why note "She or he" :-) ?